client/components/social-buttons/facebook.js

```diff
@@ -75,7 +77,7 @@ export default class FacebookLoginButton extends Component {
 
 		// Handle click async if the library is not loaded yet
 		// the popup might be blocked by the browser in that case
-		this.loadFacebookAuth().then( FB => {
+		this.initialize().then( FB => {
```

Comment:
That's probably a nitpick, but I don't really like the `this.intialize()` name,
I would prefer something like `this.getFacebookApi().then( api => api.login( ... ) )`
something like `getFacebookApi` makes more sense to me than the generic `initialize`